src/python/requirements/requirements.go

```diff
@@ -0,0 +1,91 @@
+package requirements
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+type Reqs struct{}
+
+func (m Reqs) FindAnyPackage(buildDir string, searchedPackages ...string) (bool, error) {
+	reqPackages, err := parseRequirementsWithoutVersion(filepath.Join(buildDir, "requirements.txt"))
+	if err != nil {
+		return false, err
+	}
+
+	for _, searchedPackage := range searchedPackages {
+		if containsPackage(reqPackages, searchedPackage) {
+			return true, nil
+		}
+	}
+
+	return false, nil
+}
+
+func (m Reqs) FindStalePackages(oldRequirementsPath, newRequirementsPath string, excludedPackages ...string) ([]string, error) {
+	var stalePackages []string
+
+	oldPkgs, err := parseRequirements(oldRequirementsPath)
+	if err != nil {
+		return nil, err
+	}
+
+	newPkgs, err := parseRequirements(newRequirementsPath)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, oldPkg := range oldPkgs {
+		if !containsPackage(newPkgs, oldPkg) && !packageIsExcluded(excludedPackages, oldPkg) {
```

Comment:
It should only compare names here (ignoring the version constraint) as pip-pop does: https://github.com/heroku-python/pip-pop/blob/master/bin/pip-diff#L90